guide/getting-started/visual-language.md

```diff
@@ -17,10 +17,11 @@ A brief history of bitcoin "branding" and the visual culture around bitcoin.
 
 A good addition would be a summary of the basic brand elements (color, symbol...).
 
-Illustration sources
+Source for the header image:
+https://www.dropbox.com/sh/zqnc7dbg8j5as8d/AAACidCqqJ_R11_jl0-HcA4ta?dl=0
 
-- https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=887%3A151
-- https://www.dropbox.com/sh/zqnc7dbg8j5as8d/AAACidCqqJ_R11_jl0-HcA4ta?dl=0
+Source for images used on the page:
+https://www.figma.com/file/ceVqp2xyQttnGc2h0AbZ8g/Bitcoin-Visual-Language?node-id=191%3A233&t=zLzHdkwveWgvIbRs-1
```

Comment:
```suggestion
- https://www.figma.com/file/ceVqp2xyQttnGc2h0AbZ8g/Bitcoin-Visual-Language?node-id=191%3A233&t=zLzHdkwveWgvIbRs-1
- https://www.figma.com/file/3RmrEH8560q4IgzVsvDm65/Bitcoin-Visual-Language-SBD?node-id=0%3A1&t=y51xXOeaaL1Y2JWB-1
```

Several of the graphics I added last year aren't in that Figma file, so I added my Figma file to the list. Or we could consolidate them into a single Figma file.